wizard/src/utils/detect_location.ts

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { DetectedDetails, ConfidenceLevel } from "../types";
+
+enum LocationType {
+  // None indicates a non-Location Type.
+  None = "None",
+  LatLon = "LatLon",
+  Country = "Country",
+  State = "State",
+  Province = "Province",
+  Municipality = "Municipality",
+  County = "County",
+  City = "City",
+}
+
+enum CountryFormat {
+  Name =  "Full Name",
+  ISO =  "ISO Code",
+  Alpha2 = "Alpha 2 Code",
+  Alpha3 = "Alpha 3 Code",
+  Number = "Number",
+}
+
+class DetectedLocationDetails implements DetectedDetails {
```

Comment:
If you don't intend to have functions for this class, can simply use the original interface and init  with {a:a1, b:b1, c:c1}. Then no need to create a new class.

Reply:
done. thanks